staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2226,6 +2226,10 @@ type KubeVirtConfiguration struct {
 	HandlerConfiguration           *ReloadableComponentConfiguration `json:"handlerConfiguration,omitempty"`
 	TLSConfiguration               *TLSConfiguration                 `json:"tlsConfiguration,omitempty"`
 	SeccompConfiguration           *SeccompConfiguration             `json:"seccompConfiguration,omitempty"`
+
+	// VMStateStorageClass is the name of the storage class to use for the PVCs created to preserve VM state, like TPM.
+	// The storage class must support RWX in filesystem mode.
```

Comment:
Are there requirements for the binding mode `WaitForFirstConsumer` vs. `Immediate`? Or it will work with any mode?

Reply:
I think it should work in any mode, as we don't need to mess with the PVC prior to pod creation.

Reply:
Agree with @jean-edouard, it shouldn't make any differences the PVC needs to be bound when we deploy the pod